pyseir/inference/model_fitter.py

```diff
@@ -240,36 +240,47 @@ def set_inference_parameters(self):
             self.fit_params["hosp_fraction"] = 1
 
         if len(self.fips) == 5:
-            OBSERVED_NEW_CASES_GUESS_THRESHOLD = 2
-            idx_enough_cases = np.argwhere(
-                np.cumsum(self.observed_new_cases) >= OBSERVED_NEW_CASES_GUESS_THRESHOLD
-            )[0][0]
-            initial_cases_guess = np.cumsum(self.observed_new_cases)[idx_enough_cases]
-            t0_guess = list(self.times)[idx_enough_cases]
-
             state_fit_result = load_inference_result(fips=self.state_obj.fips)
-            self.fit_params["t0"] = t0_guess
+            T0_LEFT_PAD = 5
+            T0_RIGHT_PAD = 30
+
+            # See if we can do better based on the actual data
+
+            # Let's look at the time when we've got a fifth reported case in a fips
+            OBSERVED_CUMULATIVE_MINIMUM = 2
+            cumulative_cases = np.cumsum(self.observed_new_cases)
+
+            idxs_above_threshold = np.flatnonzero(cumulative_cases >= OBSERVED_CUMULATIVE_MINIMUM)
+            if len(idxs_above_threshold) == 0:
+                # Cumulative didn't reach cutoff, use state values
+                self.fit_params["t0"] = state_fit_result["t0"]
+                self.fit_params["limit_t0"] = [
+                    state_fit_result["t0"] - T0_LEFT_PAD,
+                    state_fit_result["t0"] + T0_RIGHT_PAD,
+                ]
+            else:
+                idx_start = idxs_above_threshold[0]
+                self.fit_params["log10_I_initial"] = np.log10(
```

Comment:
How are you initializing this parameter when the state level results are used for initialization?

Reply:
I don't understand the question. There are class defaults, which can be overwritten by that hand-tuned csv, which can be overwritten in the counties here. This isn't changing any current logic.